### DISCUSSION TITLE
What is this vertical 3 dots UI element [⋮] used for?

### DISCUSSION THREAD

**Discussion by carcinocron** [reorproject/reor#123] [Q&A] [answered]
[

https://github.com/reorproject/reor/assets/4094542/81a8e4c5-1494-4e1f-99a7-8300eb58a3b6

](url)

Dragging it into files/folders produced this error: 

```
Path does not exist: Change 1
Error moving file or directory: Error: Source path does not exist.
    at moveFileOrDirectoryInFileSystem (/Users/forge/code/reor/dist-electron/main/index.js:8619:13)
```

**Comment by samlhuillier** [comment, marked as answer]
Yes thanks for reporting and sorry for the delay in responding! We are right now upgrading the editor we use to get rid of these three dots and provide a much improved experience. It should be released in the next day or so!

**Comment by samlhuillier** [comment]
This is released! Will close this discussion.
